From: Ian Campbell <ijc@debian.org>
Subject: Do not clean (remove) /debian/ dir
Date: Tue, 10 Feb 2015 03:51:27 +0000
Forwarded: not-needed

When running "make clean" in the root of an actual Debian source package we
don't want to remove the debian dir.

--- a/scripts/package/Makefile
+++ b/scripts/package/Makefile
@@ -90,8 +90,6 @@
 	$(MAKE) KBUILD_SRC=
 	$(call cmd,builddeb)
 
-clean-dirs += $(objtree)/debian/
-
 
 # tarball targets
 # ---------------------------------------------------------------------------
